styles/globals.css

```diff
@@ -47,6 +47,14 @@ div:hover > .downarrow {
   opacity: 90%;
 }
 
+.opacitykid:hover > .numberop {
```

Comment:
I do not like the name opacitykid, even though it's quite funny
numberop is not descriptive either. Rename or add a comment, i suggest giving it a better name.
There is no clue it is related to the floor-selection in the pateter page if you're only looking at the CSS-file.